app/views/shared/_parts_navigation.html.erb

```diff
@@ -1,20 +1 @@
-<% if show_alternate_guide_chapters? %>
-  <%= render "components/contents-list", aria_label: navigation_label, contents: content_item.part_link_elements %>
-<% else %>
-  <nav role="navigation" class="grid-row part-navigation" aria-label="<%= navigation_label %>" data-module="track-click">
```

Comment:
Are there any styles that can be removed?

Reply:
Not anymore, since we're keeping the old style nav for travel advice.